src/stravalib/model.py

```diff
@@ -652,9 +650,6 @@ class Lap(
     strava_model.Lap,
     BoundClientEntity,
 ):
-    # Field overrides from superclass for type extensions:
-    activity: Optional[MetaActivity] = None
-    athlete: Optional[Athlete] = None
```

Comment:
These should remain here, as they're _our_ `MetaActivity` and `MetaAthlete` types (which inherit from `BoundClientEntity`)